src/style.css

```diff
@@ -1,3 +1,95 @@
+html {
+  font-family: Roboto, Arial, sans-serif;
+  font-weight: 500;
+  font-size: 12px;
+  font-style: normal;
+  line-height: 14px;
+}
+
+* {
+  box-sizing: border-box;
+}
+
 body {
   margin: 0;
+  padding: 0;
+  background-color: gray;
+}
+
+ul {
+  list-style: none;
+  padding: 0;
+  margin: 0;
+}
+
+.nav {
+  margin: 0;
+  padding: 0;
+}
+
+.header {
+  background-color: white;
+  display: flex;
+  max-width: auto;
```

Comment:
The `max-width: auto;` is not necessary here. By default, block-level elements like `div` take up the full width available to them. If you need to constrain the width, use a specific value or percentage.